### DISCUSSION TITLE
Some characters cannot be found

### DISCUSSION THREAD

**Discussion by boromyr** [WerWolv/ImHex#1801] [Get Help]
I recently downloaded imhex and tried searching for some sequences like `;DH₫u#H` (from WinMerge) but in the search panel some characters are not recognized:
https://github.com/WerWolv/ImHex/assets/52782032/00d4bc46-346a-400d-b852-6bfdc4c708d4
Where am I going wrong?

**Comment by WerWolv** [comment]
Hey, that's probably due to your choice of font. With the default fonts it uses unifont which should support everything.
